### DISCUSSION TITLE
How to add English-Chinese and Chinese-English of Google.com to Goldendict?

### DISCUSSION THREAD

**Discussion by earthmoon** [xiaoyifang/goldendict-ng#765] [Q&A]
Hello,

How can add English-Chinese and Chinese-English of Google.com to Goldendict? I want to use it in Goldendict.

Thanks.

**Comment by xiaoyifang** [comment]
try https://github.com/xinebf/google-translate-for-goldendict

**Reply by earthmoon** [reply]
> try https://github.com/xinebf/google-translate-for-goldendict

Why En-En translate can be used by 
![image](https://github.com/xiaoyifang/goldendict-ng/assets/45897471/1aaab37a-ee88-43b4-bffc-72f5162af37d)

but  Chinese-English and English-Chinese can't used by this way?
